src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionHelpers.cs

```diff
@@ -428,6 +500,14 @@ private static CodeActionKind GetCodeActionKindFromSuggestedActionCategoryName(s
                 return codeAction;
             }
 
+            if (codeAction is FixAllCodeAction)
+            {
+                if (string.Format(FeaturesResources.Fix_All_0, currentTitle) == goalTitle)
```

Comment:
this seems *really* flakey.  especially as currentTitle is a thing you're mutating in unsafe ways.